resources/checks/A002_pgversion.sh

```diff
@@ -0,0 +1,8 @@
+# Collect postgres version info
+ssh ${HOST} "${_PSQL} -f - " <<SQL
+select
+    json_build_object('version', version(),
+        'server_version_num', current_setting('server_version_num'),
+        'server_major_ver', SPLIT_PART(current_setting('server_version'), '.', 1),
+        'server_minor_ver', SPLIT_PART(current_setting('server_version'), '.', 2));
```

Comment:
For 9.6 and older versions, this will give wrong results, here is an example fo 9.6.11:
```
test=# select  SPLIT_PART(current_setting('server_version'), '.', 2);
 split_part
------------
 6
(1 row)
```